src/DoctrineAuditBundle/Helper/AuditHelper.php

```diff
@@ -215,6 +215,7 @@ public function summarize(EntityManager $em, $entity = null, $id = null): ?array
         $meta = $em->getClassMetadata(\get_class($entity));
         $pkName = $meta->getSingleIdentifierFieldName();
         $pkValue = $id ?? $this->id($em, $entity);
+        if (is_null($pkValue)) return null; // An added guard for proxies that fail to initialize.
```

Comment:
I do not like `if` statements without curly braces. As I read somewhere 
> There are places where you can, but you **never** should.
> **Explicit** is always better than implicit.

Reply:
Done.